src/main/java/com/heycar/pairingsession/controller/ListingsController.java

```diff
@@ -0,0 +1,56 @@
+package com.heycar.pairingsession.controller;
+
+import com.heycar.pairingsession.service.ListingsService;
```

Comment:
I would like see them come up with the package structure themselves, maybe put all the classes together in the pairsession package so they can move them

Reply:
👍  will do